Automattic/crowdsignal-ui
# Update Theme Compatibility base styles

## Summary

The main purpose of this PR is to fix styling differences between the confirmation page on the Editor and on the Project Renderer. [Basically those changes](https://github.com/Automattic/crowdsignal-ui/compare/update/theme-compatibility?expand=1#diff-a1a9627a4c8a40a4a0cb7f025026f2fe11ab3aeda3d53a4e361a336439f3f8b5)

But I also used this PR to normalize most of our margins definitions to rely on a few CSS variables.
The idea is to make the margins more consistent.
Based on a common used WP variable: `--wp--style--block-gap` I created 3 Crowdsignal CSS variables to be used on the APP:

* `--cs--style--block-gap`: Used on most of the blocks
* `--cs--style--block-gap-big`: 2x the normal gap. Ex: Question Blocks
* `--cs--style--block-gap-small`: 0.5x the normal gap. Ex: Question Answers

Ref: c/8T55k0Qm-tr

## Test Instructions

* Open or create a project
* Go to the preview and submit the pages until the confirmation page
* Compare the confirmation page on the Editor and on the Project Renderer, they should look similar
* Also, take a general look at the Editor and Project Renderer styles.
